src/paulscode/android/mupen64plusae/MenuListView.java

```diff
@@ -64,18 +76,22 @@ public void setMenuResource( int menuResource )
     
     public void setMenu( Menu menu )
     {
+        Context context = getContext();
         mListData = menu;
         mAdapter = new MenuListAdapter( this, menu );
         setAdapter( mAdapter );
-        setGroupIndicator( null );
+        setChoiceMode( ListView.CHOICE_MODE_SINGLE );
+        
+        // MenuListView uses its own group indicators
+        setGroupIndicator(null);
         
-        // In case we want to add custom expand/collapse views to the groups
+        // Update the expand/collapse group indicators as needed
         setOnGroupExpandListener( new OnGroupExpandListener()
         {
             @Override
             public void onGroupExpand( int groupPosition )
             {
-                
+                reload();
```

Comment:
Reloading updates the collapse/expand arrows, which is why this line wasn't needed before.